packages/design-system-tokens/lib/types.ts

```diff
@@ -0,0 +1,37 @@
+import { ImportCall } from 'typescript';
+
+export type HexValue = `#${string}`;
+export type RGBValue = `rgb(${number},${number},${number})`;
+export type PxValue = `${number}px`;
+
+export type SpacingToken = {
```

Comment:
Should this be plural `SpacingTokens` since this actually represents a collection of tokens?